dev_requirements.txt

```diff
@@ -17,3 +17,5 @@ wheel
 pur
 
 pylint
+
+ccxt==1.79.91   # use to run test using a specific version of ccxt
```

Comment:
I think we should prefer `ccxt>=1.79.91`

Reply:
Tests are not passing if we do this due to changes in bybit (and we are currently using this exact version on octobot)

Reply:
Ok, I think we'll have to do this to prevent incompatibility in the future

Reply:
yep, especially at the moment when we can't update ccxt